yard/data/extractors/navani.yml

```
@@ -0,0 +1,36 @@
+---
+id: >-
+    navani
+name: >-
+    navani
+description: >-
+    Module for processing and plotting electrochemical data from battery cyclers.
+    Contains functions to extract dQ/dV.
+supported_filetypes:
+    - id: biologic-mpr
+    - id: neware-nda
+    - id: landt-excel
+    - id: arbin-excel
+    - id: arbin-res
+    - id: ivium-txt
```

```suggestion
    Module for processing and plotting electrochemical data from battery cyclers.
    Contains functions to extract dQ/dV. Also supports Excel and text outputs from
    Arbin and Landt.
supported_filetypes:
    - id: biologic-mpr
    - id: neware-nda
    - id: arbin-res
```

Refactoring this slightly, will make a separate PR for the arbin-res "format"